### ISSUE TITLE
Web Analytics

### ISSUE THREAD

**Issue by malyda** [XMDGCZ/XMDGCZ.github.io#12]
Do we want some kind of web analytics like Google Analytics?

**Comment by madrvojt** [comment]
Yep, I agree! Please make research in this area, which tools is best for us :)
